cli/cli-interpreter/src/test/java/org/infinispan/cli/interpreter/SiteStatementTest.java

```diff
@@ -55,7 +55,7 @@ public void testSiteStatus() throws Exception {
       String nycCache = cache("NYC", 0).getName();
       String nycSessionId = nycInterpreter.createSessionId(nycCache);
 
-      assertInterpreterOutput(lonInterpreter, lonSessionId, "site --status NYC;", "online");
+      assertInterpreterOutput(lonInterpreter, lonSessionId, "site --status \"NYC\";", "online");
```

Comment:
We should modify this test, or add a new one, that tests against a site name with an actual hyphen in it.